stdlib/http/src/main/ballerina/src/http/service_endpoint.bal

```diff
@@ -177,11 +177,13 @@ public type ServiceEndpointConfiguration record {|
 # should successfully be authorozed.
 # + positiveAuthzCache - The caching configurations for positive authorizations.
 # + negativeAuthzCache - The caching configurations for negative authorizations.
+# + position - The authn/authz filter position of the filter array. The position values starts from 1 and it is set to 1 implicitly.
```

Comment:
IMO, this statement gives misleading idea. Ballerina array starts with 0. So in here we say,  we are having filter array, but position start with 1. Can't we align with that normal array positioning?

Reply:
Yes. Agree on this. It is fixed with https://github.com/ballerina-platform/ballerina-lang/pull/17124/commits/31c544eea1feb656ae2af1a21d0c4e4103057aa3